uaa-server/uaa.yml

```diff
@@ -105,7 +104,6 @@
         authorities: uaa.resource
         authorized-grant-types: authorization_code
         redirect-uri: http://localhost:8082/login
-        scope: openid,menu.read,order.me
```

Comment:
I noticed this line was removed. If that's intentional, feel free to resolve and merge.

Reply:
Good catch, I must have lost this when applying suggestions yesterday